src/connection_pool.rs

```diff
@@ -82,7 +82,7 @@ impl<C, R, F> ConnectionPool<C, R, F>
 where
     C: Connection,
     R: Resolver,
-    F: FnMut(&Backend) -> C + Send + Sync + 'static
+    F: FnMut(&Backend) -> C + Send + 'static
```

Comment:
The 'static in the return implies the backend will be present for the lifetime of the connection pool?

Reply:
This implies that the `FnMut`, `F`, will be present for the lifetime of the pool which makes sense I suppose since the pool is reliant on it to do anything useful.